@kiva/kv-components/vue/KvSideSheet.vue

```diff
@@ -184,8 +200,16 @@ export default {
 							transition: 'all 0.5s ease-in-out',
 						};
 					}, 10);
-				} else {
-					modalStyles.value = {};
+				} else if (stickyControls.value) {
+					let sheetHeight = 100;
+					const navElement = document.querySelector('nav'); // Check if header element exists
+					if (window) {
+						sheetHeight = window.innerHeight - (navElement?.offsetHeight ?? 0);
```

Comment:
What about when the page scrolls? The side sheet will no longer want the `offsetHeight` but whatever part of the nav is visible subtracted.